beater/request/context.go

```diff
@@ -79,7 +79,9 @@ type Context struct {
 	// UserAgent holds the User-Agent request header value.
 	UserAgent string
 
-	w             http.ResponseWriter
```

Comment:
I think it's fine to export the ResponseWriter (or add methods that call through to the ResponseWriter's?), but if we do that I'd like to clarify the existing `Context.Write` method by renaming it to `Context.WriteResult`. Also, it would be good to add some comments that only one or the other should be used, and not both.

Reply:
Renamed the exported `ResponseWriter` and renamed `Context.Write` to `Context.WriteResult` in 8fedc5199 and 5fdff5d11